tools/run_tests/run_tests.py

```diff
@@ -425,7 +425,7 @@ def make_options(self):
     return []
 
   def build_steps(self):
-    return [['tools/run_tests/build_python.sh', tox_env] 
+    return [['tools/run_tests/build_python.sh', tox_env]
```

Comment:
Reverted. Thanks a lot for the review! It seems my auto format tool deleted this blank.